### ISSUE TITLE
const enum: supress value comment in emitted javascript

### ISSUE THREAD

**Issue by stanthomas** [Microsoft/TypeScript#1833]
Propose that the value comment associated with const enums be supressed when using "removeComments" flag.

TypeScript 1.4:
```javascript
const enum color {red,  green, blue}
var c = color.blue;
```

Emitted javascript (TS 1.4):
```javascript
var c = 2 /* blue */;
```

Change so that, with "removeComments" flag, emits:
```javascript
var c=2;`
```

This is consistent with the goal of reducing the size of the generated code (although I'll run the javascript thru the Closure Compiler for release) and it is odd to see these enum values stand out as the only comments in the javascript.

**Comment by NN---** [comment]
:+1: removeComments should remove comments.

**Comment by RyanCavanaugh** [comment]
Should be a simple fix if someone wants to send a PR over

**Comment by DanielRosenwasser** [comment]
:-1: This would bring unnecessary complexity and the comments are useful for debugging. If, as you said, you're going to run it through something that's going to strip out the comments, why does it matter?

**Comment by RyanCavanaugh** [comment]
I don't think it's unreasonable or complex that emitter shouldn't *add* new comments when you've passed the (already-supported) `--removeComments` flag.

**Comment by DanielRosenwasser** [comment]
My apologies, I thought that the `removeComments` flag was proposed. If we already have it in place, it makes sense in that context. :+1:

**Comment by stanthomas** [comment]
pull request submitted

**Comment by mhegazy** [comment]
Fixed by #1855
